docs/self-managed/setup/deploy/local/c8run.md

```diff
@@ -98,6 +96,55 @@ To add a custom Connector:
 
 Once configured correctly, your Connectors are available for use in Modeler.
 
+## Use Camunda APIs
+
+Camunda 8 Run authenticates with the [Tasklist](/apis-tools/tasklist-api-rest/tasklist-api-rest-overview.md), [Operate](/docs/apis-tools/operate-api/overview.md), and [Zeebe](/apis-tools/zeebe-api/grpc.md) APIs, as well as the unified [Camunda 8 REST API](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md), by including cookie headers in each request. This cookie can be obtained by using the API endpoint `/api/login`.
+
+To authenticate and begin making requests, take the following steps:
+
+<Tabs groupId="api" defaultValue="v1" queryString values={
+[
+{label: 'Tasklist, Operate, and Zeebe', value: 'v1' },
+{label: 'Camunda 8 REST API', value: 'v2' },
+]}>
+
+<TabItem value='v1'>
+
+1. Log in as user 'demo' and store the cookie in the file `cookie.txt`:
+
+```shell
+curl -c cookie.txt -X POST 'http://localhost:8080/api/login?username=demo&password=demo'
```

Comment:
I see inconsistent flags (`-c`, `-X`, `-b`), which are optional AFAIK, but the calls are successful. I would prefer we remove flags for clarity unless we need them. WDYT @pepopowitz?

Reply:
+1 to consistency across the calls. 

There are a couple other things that I think help readability on example curl requests, which I tried to implement on the auth rewrites ([example](https://docs.camunda.io/docs/apis-tools/tasklist-api-rest/tasklist-api-rest-authentication/?environment=saas)): 

1. One argument per line, using `\` to tell the shell that more is coming
2. Long-form flags (e.g. `--cookie-jar` instead of `-c`) because seeing the full name can help people who don't know what the short-form translates to (read: me 😅)

So in this example, something like: 

```sh
curl --request POST 'http://localhost:8080/api/login?username=demo&password=demo' \
  --cookie-jar cookie.txt 
```

In the next example, something like: 
```sh
curl --request POST 'http://localhost:8080/v1/process-definitions/search'  \
  --cookie cookie.txt
  --header 'Content-Type: application/json' \
  --data-raw '{}`
```

Reply:
Thank you! I've gone ahead and updated this section to use the clearer formatting and flags.